src/components/metrics/index.tsx

```diff
@@ -58,6 +58,15 @@ export const options = {
       grid: {
         display: false,
         drawBorder: false,
+      },
+      ticks: {
+        // For a category axis, the val is the index so the lookup via getLabelForValue is needed
+        // @ts-ignore
+        callback: function(val:any, index:any) {
```

Comment:
ts-ignore can be removed - don't need it with _any_  variable type. "index" variable is not used.
```
        callback: function(val: any) {
          if (val === 0) return "";
          return val;
        }
```

Reply:
ok sure thing